### PR TITLE
Fix typos discovered by codespell

### PR DESCRIPTION
**Pull request by cclauss**
https://pypi.org/project/codespell/

### CODE REVIEW

File: upgrading/earlier/whats-new-2.0.rst [hunk 1]
```diff
@@ -495,7 +495,7 @@ Other bug fixes
 *  Isolate non-standard attributes on MathML elements so that they don't
    interfere with the inner workings of MathJax.
 
-*  Properly handle width of border and padding in merrors in `HTML-CSS`
+*  Properly handle width of border and padding in mirrors in `HTML-CSS`
```
Comment (cclauss) [inline upgrading/earlier/whats-new-2.0.rst:498]:
```suggestion
*  Properly handle width of border and padding in errors in `HTML-CSS`
```

File: input/tex/extensions/ams.rst [hunk 2]
```diff
@@ -67,12 +67,12 @@ following values:
 .. _tex-ams-multlineWidth:
 .. describe:: multlineWidth: '100%'
 
-The width to use for multline environments.
+The width to use for multiline environments.
```
Comment (dpvc) [inline input/tex/extensions/ams.rst:70]: The environment name really is `multline`, not `multiline`, so this needs to be left as is.

File: input/tex/extensions/ams.rst [hunk 3]
```diff
@@ -67,12 +67,12 @@ following values:
...
 
 .. _tex-ams-multlineIndent:
 .. describe:: multlineIndent: '1em'
 
-The margin to use on both sides of multline environments.
+The margin to use on both sides of multiline environments.
```
Comment (dpvc) [inline input/tex/extensions/ams.rst:75]: Same here.

File: upgrading/earlier/whats-new-2.0.rst [hunk 4]
```diff
@@ -495,7 +495,7 @@ Other bug fixes
 *  Isolate non-standard attributes on MathML elements so that they don't
    interfere with the inner workings of MathJax.
 
-*  Properly handle width of border and padding in merrors in `HTML-CSS`
+*  Properly handle width of border and padding in errors in `HTML-CSS`
```
Comment (dpvc) [inline upgrading/earlier/whats-new-2.0.rst:498]: This comment is about the `merror` elements, so should remain as it was originally.

File: upgrading/earlier/whats-new-2.0.rst [hunk 5]
```diff
@@ -519,7 +519,7 @@ Other bug fixes
 
 *  Handle border and padding CSS styles better in `HTML-CSS` output.
 
-*  Fixed multline environment to put tags on bottom row when
+*  Fixed multiline environment to put tags on bottom row when
```
Comment (dpvc) [inline upgrading/earlier/whats-new-2.0.rst:522]: Again, this was correct originally.

File: upgrading/earlier/whats-new-2.5.rst [hunk 6]
```diff
@@ -64,7 +64,7 @@ TeX emulation
 *   `#889 <https://github.com/mathjax/MathJax/issues/889>`__ remove extra space around some parenthesis constructs.
 *   `#856 <https://github.com/mathjax/MathJax/issues/856>`__ Recognize comma as decimal delimiter in units.
 *   `#877 <https://github.com/mathjax/MathJax/issues/877>`__ Fix bug related to multiple accent having different width.
-*   `#832 <https://github.com/mathjax/MathJax/issues/832>`__ Fix multline environment not being centered in HTML-CSS output.
+*   `#832 <https://github.com/mathjax/MathJax/issues/832>`__ Fix multiline environment not being centered in HTML-CSS output.
```
Comment (dpvc) [inline upgrading/earlier/whats-new-2.5.rst:67]: Same here.

File: .github/workflows/codespell.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# https://pypi.org/project/codespell
+name: codespell
+on: [pull_request, push]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: pip install codespell
+      - run: codespell --ignore-words-list=braket,ket
```
Comment (dpvc) [inline .github/workflows/codespell.yml:10]: Can you add `multline` and `merrors` to this list to avoid the false positives?

**Review by dpvc — CHANGES_REQUESTED** [review]
Thanks for your suggestions and the workflow file.  I did note a few false positives, so could you resolve those and add them to the ignored word list?

File: input/tex/extensions/ams.rst (hunk 2 above)
Comment (cclauss) [reply input/tex/extensions/ams.rst:70]:
```suggestion
The width to use for multline environments.
```

File: input/tex/extensions/ams.rst (hunk 3 above)
Comment (cclauss) [reply input/tex/extensions/ams.rst:75]:
```suggestion
The margin to use on both sides of multline environments.
```

File: upgrading/earlier/whats-new-2.0.rst (hunk 4 above)
Comment (cclauss) [reply upgrading/earlier/whats-new-2.0.rst:498]:
```suggestion
*  Properly handle width of border and padding in merrors in `HTML-CSS`
```

File: upgrading/earlier/whats-new-2.0.rst (hunk 5 above)
Comment (cclauss) [reply upgrading/earlier/whats-new-2.0.rst:522]:
```suggestion
*  Fixed multline environment to put tags on bottom row when
```

File: upgrading/earlier/whats-new-2.5.rst (hunk 6 above)
Comment (cclauss) [reply upgrading/earlier/whats-new-2.5.rst:67]:
```suggestion
*   `#832 <https://github.com/mathjax/MathJax/issues/832>`__ Fix multline environment not being centered in HTML-CSS output.
```

File: .github/workflows/codespell.yml (hunk 7 above)
Comment (cclauss) [reply .github/workflows/codespell.yml:10]:
```suggestion
      - run: codespell --ignore-words-list=braket,ket,merror,multline
```

**Review by dpvc — APPROVED** [review]
Thanks for making the changes.  I will merge.

**Pull request merged**